LTS.md

```diff
@@ -0,0 +1,25 @@
+# Supported versions
+
+## Long Term Support
+
+Express Long Term Support (LTS) version is provided as per defined in this document - and also include all the packages part of the Express.js Organization (express, pillarjs and jshttp).
```

Comment:
Just to clarify, all packages will need to start following this and not just `express`?

Reply:
All packages would need to provide this as a minimum, I’d assume - but they could certainly support longer if desired, since it’ll probably be cheap for smaller packages to do so.

Reply:
I think it is really important we clarify this. I think I subtly disagree with @ljharb on this, but so far the docs themselves do not provide enough clarity to have that conversation imo and I think we didn't have that discussion in the other original thread either. I will open an issue in the discussions repo so we can address that. We should block merging this on resolution to that discussion.

Reply:
https://github.com/expressjs/discussions/issues/210